src/System.Security.Cryptography.Primitives/tests/CryptoStream.netcoreapp.cs

```diff
@@ -64,6 +64,31 @@ public static void DisposeAsync_DerivedStream_InvokesDispose()
             }
         }
 
+        [Theory]
+        [InlineData("Aes")]
+        [InlineData("Rijndael")]
+        public static void PaddedAesRijdnael_PartialRead_Success(string algName)
+        {
+            SymmetricAlgorithm alg = SymmetricAlgorithm.Create(algName);
```

Comment:
This is a disposable resource, please put it in a using.

Reply:
Done